src/ucp/wireup/wireup.h

```diff
@@ -180,6 +180,21 @@ static inline int ucp_worker_is_tl_2iface(ucp_worker_h worker,
               UCT_IFACE_FLAG_CONNECT_TO_IFACE);
 }
 
+/**
+ * Check if TL supports only point to pont connections.
+ *
+ * @param [in]  worker       UCP worker.
+ * @param [in]  rsc_index    resource index.
+ *
+ * @return 1 if TL supports only point to pont connections, otherwise 0.
+ */
+static inline int ucp_worker_is_tl_p2p_only(ucp_worker_h worker,
+                                            ucp_rsc_index_t rsc_index)
+{
+    return ucp_worker_is_tl_p2p(worker, rsc_index) &&
+          !ucp_worker_is_tl_2iface(worker, rsc_index);
+}
```

Comment:
why not just use !ucp_worker_is_tl_2iface()?
maybe we can filter/assert on TLS which have neither 2ep nor 2iface during ucp worker create